src/main/java/domain/Race.java

```diff
@@ -2,42 +2,84 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 public class Race {
 
     public static List<RacingCar> racingCarList = new ArrayList<>();
 
-    public static void createCar(int carCount) {
-        for (int i = 0; i < carCount; i++) {
-            RacingCar car = new RacingCar();
+    public static void createCars(RacingCarNameRequest nameList) {
```

Comment:
```suggestion
    public static void createCars(RacingCarNameRequest names) {
```
변수명에 타입을 선언하는것보단 복수형으로 선언하는것이 어떨까요 ?
복수형으로 선언하는것에 대한 이유를 고민해보시면 좋을것 같아요 😄

Reply:
@sah3122 미처 고민해보지 못했던 부분 짚어주셔서 감사합니다 :) 타입보다 복수형이 나은 이유는 아래와 같습니다.

1. `nameList`를 쓰면 이 객체가 리스트나 배열같이 컬렉션인 것처럼 보이지만, 실상은 `RacingCarNameRequest`라는 객체입니다. 따라서 혼동을 피하기 위해 List를 빼는 게 좋습니다.
2. 해당 객체에서 우리가 꺼내고자 하는 것이 이름인데, `names`라는 복수형을 쓰면 이름이 여러 개라는 것을 짐작할 수 있습니다.

Reply:
개인적으로 변수명에 List와 같은 타입을 선언하지 않는 이유로 아래와 같은 경우도 포함됩니다.
예를 들어 List로 선언한 변수를 Set Collection으로 변경하는 경우 많은 코드가 수정되어야 하고 때에 따라선 의미가 완전히 바뀔수 있다고 생각합니다 😄